### ISSUE TITLE
Apollo should recognize jbrowse dot notation

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#1839]
FYI @childers  Let me know if this is correct.

**Comment by childers** [comment]
@nathandunn Do you have some examples or a description of what the JBrowse dot notation looks like?  I'm reading about dot notation in general, but would rather be clear as to what you are referring to before answering. In general, the parameters getting processed during login, authenticated (and possibly logout) events are not being handled correctly in the URL param format.

The closed issue #1840 has some additional information and server log entries if that helps.  Otherwise, just let me know what information you need and I'll pull it together.

**Comment by nathandunn** [comment]
@childers     This is the library I am using.   https://github.com/rhalff/dot-object

Keep in mind there are some simplifications as well.  

Also, I was wrong earlier about the `addStores`.  It documented it correctly, but I'm not sure why I made the implementation `addStore`.  I've fixed it on the JBrowse end.   There are a couple of things we need to fix on the Apollo end on #1842 that @deepakunni3  is looking into.
